src/libstd/sys/common/unwind/gcc.rs

```diff
@@ -231,3 +231,31 @@ pub mod eabi {
         }
     }
 }
+
+#[cfg(all(target_os="windows", target_arch = "x86", target_env="gnu", not(test)))]
+#[lang = "eh_unwind_resume"]
+#[unwind]
+unsafe extern fn rust_eh_unwind_resume(panic_ctx: *mut u8) -> ! {
+    uw::_Unwind_Resume(panic_ctx as *mut uw::_Unwind_Exception);
+}
+
+#[cfg(all(target_os="windows", target_arch = "x86", target_env="gnu"))]
+pub mod eh_frame_registry {
```

Comment:
Can you be sure to add a comment here pointing to a location which documents what this all is?